modules/apps/export-import/export-import-report-test/src/testIntegration/java/com/liferay/exportimport/report/internal/exception/handler/test/ImportStagedModelExceptionHandlerTest.java

```diff
@@ -139,14 +171,50 @@ private void _test(long expectedGroupId, String expectedScope, Group group)
 			serviceRegistration.unregister();
 		}
 
+		BackgroundTask backgroundTask =
```

Comment:
I don't think REST dependencies should be mixed in the service tests. Also, is there a specific reason for this, because checking the number of entries increase is already covered? If it needs an extra assertion, I think it should use the service directly and not the REST endpoint. Alternatively, maybe creating a new test in the REST tests, but I'm not sure if that would be necessary in this case.

Reply:
I agree that I'm mixing tests but the reason were:

* I have included it here because this is where the service is tested when there is already another transaction. The test checks whether errors are stored in the database, but not whether they are indexed.
* The search logic could be included, but since I did not include business logic in the test, I preferred to call the API, which is what is used from the UI.
* This is a case that only occurs in classic Staging, so this test can be deleted entirely when the migration is complete.

If it is considered unnecessary, I can delete it.